src/faaslet/Faaslet.cpp

```diff
@@ -20,16 +21,51 @@
 using namespace isolation;
 
 namespace faaslet {
-void Faaslet::postFlush()
+
+std::mutex flushMutex;
+
+void preloadPythonRuntime()
+{
+    auto logger = faabric::util::getLogger();
+
+    auto conf = faabric::util::getSystemConfig();
+    if (conf.pythonPreload != "on") {
+        logger->info("Not preloading python runtime");
+        return;
+    }
+
+    logger->info("Preparing python runtime");
+
+    faabric::Message msg =
+      faabric::util::messageFactory(PYTHON_USER, PYTHON_FUNC);
+    msg.set_ispython(true);
+    msg.set_pythonuser("python");
+    msg.set_pythonfunction("noop");
+    faabric::util::setMessageId(msg);
+
+    faabric::scheduler::Scheduler& sch = faabric::scheduler::getScheduler();
+    sch.callFunction(msg, true);
+}
+
+void Faaslet::flush()
```

Comment:
This `flush` function is basically the same, but flushes cached function files and kills the Faaslet itself at the end (to force everything to reload).